packages/platform/types/event-types/event-types_2024_06_14/outputs/event-type.output.ts

```diff
@@ -419,8 +419,56 @@ export class TeamEventTypeOutput_2024_06_14 extends BaseEventTypeOutput_2024_06_
   @DocsProperty()
   hideCalendarEventDetails?: boolean;
 
+  @IsOptional()
+  @ValidateNested()
+  @Type(() => EventTypeTeam)
+  @DocsProperty()
+  team?: EventTypeTeam | null;
```

Comment:
In the output.service.ts we have
```
team: {
        id: teamId,
        name: databaseEventType?.team?.name,
```
but in this output we have `@IsOptional()` and team is marked as optional `team?` but it looks like the team is always returned.

Reply:
actually yes you're right the team is always returned need to fix this